docs/sandbox-doc-primer.md

```diff
@@ -0,0 +1,213 @@
+
+When writing for new users (or indeed, any user who isn't part of the
+contributor community), try not to assume that the user knows anything about the
+product. This is especially true when writing task documentation.
+
+In the database example, say that configuring the server consists of copying the
+config file template to the config directory, setting permissions on the file,
+and editing the file to include the server IPs and some other parameters. Time
+and again we've seen configuration documentation that consists only of which
+config parameters to set. An effective configuration task, though, would walk
+through these steps:
+
+1. Copy the template from folder X to folder Y.
+1. Name the file Z.
+1. Set permissions on the file to ABC.
+1. Edit the file to set parameters Q, R, S and T. (And include a link to the
+   config reference\!)
+
+Don't worry about giving too much information. Novice users need it, and expert
+users will ignore it.
+
+### Support the task documentation
+
+Finally, support the task documentation with conceptual and reference
+documentation. The developer, for example, will need a complete reference for
+the REST API. The admin will need a complete description of every server
+configuration option. Both will need a reference for the CLI, if there is one.
+
+Conceptual information should always include a technical overview. This includes
+the product's purpose, architecture, and design rationales. This helps
+prospective users decide whether it's a solution for their problem, and helps
+advanced users reason about how to use or even extend its advanced features.
+
+## Nuts and bolts: deliverables and infrastructure
+
+### Required documentation
+
+A minimal documentation set varies by product, but some documents are (almost)
+always required to make the product usable:
+
+- A technical overview
+- A getting started guide
+- Reference manual(s)
+
+### Where to put it
+
+At the sandbox stage, making the documentation available in some form is more
+important than hosting it in any particular place. That said, CNCF TechDocs has
+a [number of resources](https://github.com/cncf/techdocs/tree/main/docs) for
+creating web-based documentation.
+
+Even if you don't use the CNCF recommended tools, it's worth reading the
+recommendations to preview the sorts of issues you'll encounter as the
+documentation matures. These issues parallel those facing the code
+infrastructure, and include things like scaling, hosting, versioning, indexing,
+SEO, and finding contributing writers.
+
+### What’s next
+
+For a thorough preparation in working with technical documentation, the
+following resources are available:
+
+[Open Source Documentation Essentials](https://training.linuxfoundation.org/training/open-source-technical-documentation-essentials-lfc111/)  
+[Creating Effective Documentation for Developers](https://training.linuxfoundation.org/training/creating-effective-documentation-for-developers-lfc112/)  
```

Comment:
@chalin, the linter is barking at these two lines (too long). How should we handle long lines (or how can I tell the linter it's OK in this case?)

Reply:
@nate-double-u @chalin Problem seems to have been spaces at the ends of the lines. These two lines seem to be behaving now (the rule seems to be that links are exceptions to the 80-character line length limit). However, there's another formatter failure now caused by a Prettier violation. Is there a way to get Prettier to give more informative error messages?